### PR TITLE
Add API tests

### PR DESCRIPTION
**Pull request by mbreza**

### CODE REVIEW

**Comment by vvisnia** [comment]
"No description provided." HOW RUDE!

File: src/test/java/apitests/FreightTransportTest.java [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+				vehicleList, addressList.get(1), addressList.get(0))));		
+		
+	}
+
+	@After
+	public void cleanup() throws Exception {			
+		
+		if(FreightTransportList.size()!=0){
+			
+			FreightTransportList.remove(1);
```
Comment (m-ciesielski) [inline src/test/java/apitests/FreightTransportTest.java]: Why first element of `FreightTransportList` is not deleted during cleanup?

**Comment by vvisnia** [comment]
You also have to resolve conflicts b4 merging.

**Comment by vvisnia** [comment]
Do not merge. Stranger errors.!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Pull request closed without merging**